edu.wpi.first.wpilib.plugins.cpp/resources/templates/command-based/Robot.cpp

```diff
@@ -13,7 +13,8 @@
 class Robot: public frc::IterativeRobot {
 public:
 	void RobotInit() override {
-		chooser.AddDefault("Default Auto", new ExampleCommand());
+		defaultAuto.reset(new ExampleCommand());
+		chooser.AddDefault("Default Auto", defaultAuto.get());
 		// chooser.AddObject("My Auto", new MyAutoCommand());
```

Comment:
Add an instance variable for this line too, since it would be a memory leak if uncommented.